csharp/tools/tracing-config.lua

```diff
@@ -31,7 +31,8 @@ function RegisterExtractorPack(id)
             local firstCharacter = string.sub(arg, 1, 1)
             if not (firstCharacter == '-') and not (firstCharacter == '/') then
                 Log(1, 'Dotnet subcommand detected: %s', arg)
-                if arg == 'build' or arg == 'msbuild' then match = true end
+                if arg == 'build' or arg == 'msbuild' or arg == 'publish' or arg == 'pack' or arg == 'test' or
+                    arg == 'run' then match = true end
```

Comment:
Minor: update the comment above to be more generic. Currently it only mentions `build`.

Reply:
Done.